_docs/tasks/traffic-management/ingress.md

```diff
@@ -269,107 +214,78 @@ rules.
-     name: foo-service-route
-   spec:
-     destination:
-       name: ingress-dummy-service
-     match:
-       request:
-         headers:
-           uri:
-             prefix: "/foo.FooService"
-     precedence: 1
-     route:
-     - weight: 100
-       destination:
-         name: foo-service
-   ---
-   apiVersion: config.istio.io/v1alpha2
-   kind: RouteRule
-   metadata:
-     name: bar-service-route
+     name: status-route
    spec:
      destination:
-       name: ingress-dummy-service
+       name: httpbin
      match:
+       # Optionally limit this rule to istio ingress pods only
+       source:
+         name: istio-ingress
+         labels:
+           istio: ingress
        request:
          headers:
            uri:
-             prefix: "/bar.BarService"
-     precedence: 1
+             prefix: /delay/ #must match the path specified in ingress spec
+                 # if using prefix paths (/delay/.*), omit the .*.
+                 # if using exact match, use exact: /status
      route:
      - weight: 100
-       destination:
-         name: bar-service
-   EOF      
+     httpReqTimeout:
+       simpleTimeout:
+         timeout: 4s
+  EOF
    ```
 
+If you were to make a call to the ingress with the URL
+`http://$INGRESS_HOST/delay/10`, you will find that the call returns in 4s
+instead of the expected 10s delay.
+
+You can use other features of the route rules such as redirects, rewrites,
+routing to multiple versions, regular expression based match in HTTP
+headers, websocket upgrades, timeouts, retries, etc. Please refer to the
+[routing rules]({{home}}/docs/reference/config/traffic-rules/routing-rules.html)
+for more details.
+
+> Note 1: Fault injection does not work at the Ingress
+
+> Note 2: When matching requests in the routing rule, use the same exact
+> path or prefix as the one used in the Ingress specification.
 
 ## Understanding ingresses
 
-Ingresses provide gateways for external traffic to enter the Istio service mesh
-and make the traffic management and policy features of Istio available for edge services.
+Ingresses provide gateways for external traffic to enter the Istio service
+mesh and make the traffic management and policy features of Istio available
+for edge services.
+
+The servicePort field in the Ingress specification can take a port number
+(integer) or a name. The port name must follow the Istio port naming
+conventions (e.g., `grpc-*`, `http2-*`, `http-*`, etc.) in order to
+function properly. The name used must match with the port name in the
+backend service declaration.
 
-In the preceding steps we created a service inside the Istio service mesh and showed how
-to expose both HTTP and HTTPS endpoints of the service to external traffic.
-We also showed how to control the ingress traffic using an Istio route rule.
+In the preceding steps we created a service inside the Istio service mesh
+and showed how to expose both HTTP and HTTPS endpoints of the service to
+external traffic.
```

Comment:
put back: 
We also showed how to control the ingress traffic using an Istio route rule.